may13th.ts

```
@@ -0,0 +1,103 @@
+  if (fixData.fixes.length === 0) {
+    res.status(404).json({
+      error: 'No fixes found for the snippet!'
+    })
+    return
+  }
+  res.status(200).json({
+    fixes: fixData.fixes
+  })
+}
+
+export const checkCorrectFix = () => async (req: Request<{}, {}, VerdictRequestBody>, res: Response, next: NextFunction) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)